toolkit/offchain/src/reserve/mod.rs

```diff
@@ -17,14 +19,35 @@ pub mod init;
 pub mod update_settings;
 
 #[derive(Clone, Debug)]
-pub(crate) struct ReserveData {
-	pub(crate) scripts: scripts_data::ReserveScripts,
-	pub(crate) auth_policy_version_utxo: OgmiosUtxo,
-	pub(crate) validator_version_utxo: OgmiosUtxo,
-	pub(crate) illiquid_circulation_supply_validator_version_utxo: OgmiosUtxo,
+pub struct ReserveData {
+	pub scripts: scripts_data::ReserveScripts,
+	pub auth_policy_version_utxo: OgmiosUtxo,
+	pub validator_version_utxo: OgmiosUtxo,
+	pub illiquid_circulation_supply_validator_version_utxo: OgmiosUtxo,
 }
 
-pub(crate) async fn get_reserve_data<
+impl ReserveData {
+	pub async fn get_reserve_settings<
```

Comment:
I don't know about *release* (I expect it could benefit too) but I think few places like *debit* and *handover* in the code could benefit if there was a field `reserve_utxo: OgmiosUtxo` in `ReserveData` and also a method `reserve_datum(&self) -> Option<ReserveDatum>` .

I think it will be okay if it is merged like it is now, but please expect that when three of us are done with functionality, there will be some PR with cleanup. Ok?